experiment/measurer/measure_manager.py

```diff
@@ -77,8 +78,13 @@ def measure_main(experiment_config):
     measurers_cpus = experiment_config['measurers_cpus']
     runners_cpus = experiment_config['runners_cpus']
     region_coverage = experiment_config['region_coverage']
-    measure_loop(experiment, max_total_time, measurers_cpus, runners_cpus,
-                 region_coverage)
+    local_experiment = experiment_utils.is_local_experiment()
+    if local_experiment:
+        measure_manager_loop(experiment, max_total_time, measurers_cpus,
```

Comment:
I thought we would have to wait until we had implemented pub sub queues before using this to non local experiments, but I guess we can already allow it to both local and non local, right?

Reply:
Yeah, it should be able to run in prod. But I guess the disadvantage of doing so, is that once we had pub/sub queues we would never need to run this in prod again.
Your choice, we will get better testing if we run this in prod, but it may reveal more problems than we care to fix.